herbert_core/classes/MPIFramework/@parallel_config/private/check_and_set_worker_.m

```diff
@@ -32,6 +40,23 @@
         
     end
 else % worker function is available.
-    config_store.instance().store_config(obj,'worker',new_wrkr);
+    config_store.instance().store_config(obj, 'worker', new_wrkr);
+    config_store.instance().store_config(obj, 'is_compiled', false);
 end
+end % function
 
+function out = check_compiled_(worker)
+    out = '';
+    if exist(worker, 'file')
+        % Assume if input is full path to file, then it is a compiled worker
+        out = worker;
+    else
+        if ispc(), cmd = 'where'; else, cmd = 'which'; end
```

Comment:
I personally dislike inline-`if`s like this. I would expand this out to make it clear.

Reply:
What we really need is a ternary operator...